server/models/Profile.js

```diff
@@ -0,0 +1,42 @@
+const mongoose = require('mongoose');
+
+const ProfileSchema = new mongoose.Schema(
+  {
+    firstName: {
+      type: String,
+      required: true,
+    },
+    lastName: {
+      type: String,
+      required: true,
+    },
+    description: {
+      type: String,
+    },
+    gender: {
+      type: String,
+    },
+    birthDate: {
+      type: Date,
+    },
+    email: {
```

Comment:
Please remove the email field - it already exists in the User model and we do not want to store duplicate data across our DB :)